### PR TITLE
add community.vmware

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

File: scenarios/community/vmware/ansible.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+netcommon:
```
Comment (pabelanger) [inline scenarios/community/vmware/ansible.yml]: does vmware depend on netcommon? if not, we can delete this

File: scenarios/community/vmware/vmware.yml [hunk 2]
```diff
@@ -0,0 +1,191 @@
+vmware:
```
Comment (pabelanger) [inline scenarios/community/vmware/vmware.yml]: This file should be

scenarios/community/vmware/community.yml

File: scenarios/community/vmware/vmware.yml [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+  inventory:
+  - vmware_vm_inventory.py
+  module_utils:
+  - vmware.py
+  - vmware_httpapi/VmwareRestModule.py
+  - vmware_httpapi/__init__.py
+  - vmware_rest_client.py
+  - vmware_spbm.py
+  modules:
+  - cloud/vmware/__init__.py
```
Comment (pabelanger) [inline scenarios/community/vmware/vmware.yml]: you could glob here, to reduce files if everything in folder is migrated

Comment (goneri) [reply scenarios/community/vmware/vmware.yml]: I did it, but this also create a delta with the original configuration.

**Comment by goneri** [comment]
recheck

File: scenarios/community/vmware/ansible.yml (hunk 1 above)
Comment (goneri) [reply scenarios/community/vmware/ansible.yml]: ok

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by pabelanger** [comment]
https://object-storage-ca-ymq-1.vexxhost.net/v1/a0b4156a37f9453eb4ec7db5422272df/ansible_63/63/1d405e14a1b648561bb36188b3ab08c43c7aceab/check/network-collections-migration-community-vmware/ad19a21/job-output.html#l12254

That is a result of running all yaml file into ruamel.yaml reformat script: https://github.com/ansible-network/network_collections_migration/blob/master/tools/yamlf.py

This is super hacky, and something we just did for network. You can either add logic to playbooks to skip this, or we add another fix up script to deal with this, for example: 

https://github.com/ansible-network/network_collections_migration/blob/master/tools/cleanup_nxos.sh

I would lean towards bashscript, but possible we need to hold node to inspect file to see what failed, but you also should be able to reproduce locally.

**Comment by goneri** [comment]
I will continue with manually sync for now.

**Pull request closed without merging**